src/partition/tests/ReceivedPartitionTest.cpp

```diff
@@ -1096,11 +1095,12 @@ void testParallelSetOwnerInformation(T &mesh, int dimensions)
   part.addFromMapping(boundingFromMapping);
   part.addToMapping(boundingToMapping);
 
-  part._mesh->computeBoundingBox();
-  part.prepareBoundingBox();
+  mesh->computeBoundingBox();
 
-  part.tagMeshFirstRound();
-  part.createOwnerInformation();
+  receivedPartitionFixture fixture;
+  fixture.prepareBoundingBox(part);
+  fixture.tagMeshFirstRound(part);
+  fixture.createOwnerInformation(part);
```

Comment:
If the functions are static one can do this:

```suggestion
  using Access = receivedPartitionFixture;
  Access::prepareBoundingBox(part);
  Access::tagMeshFirstRound(part);
  Access::createOwnerInformation(part);
```

Reply:
done